models/newsletter.js

```diff
@@ -0,0 +1,78 @@
+const { loggers } = require("winston");
+const firestore = require("../utils/firestore");
+const newsletterModel = firestore.collection("newsletter");
+
+const DB_ERROR = "Not able to conenct to DB";
+
+const subscribe = async (data) => {
+  try {
+    const { email } = data;
+    const check = await newsletterModel.where("email", "==", email).get();
+    let exist = false;
+    check.forEach((ch) => {
+      const data = ch.data();
```

Comment:
Can we change the argument name `ch` to something more meaningful

Reply:
done